### ISSUE TITLE
bundle includes unused items

### ISSUE THREAD

**Issue by max-pub** [denoland/deno#9546]
consider the following ```lib.js```
```js
export function a() { return 'a' }
export function b() { return 'b' }
export function c() { return b() }
```

and the following file using the above:
```js
import { a } from './lib.js'
a()
```

```deno bundle``` creates:
```js
function a() {return 'a';}
function b() {return 'b';}
a();
```

but should be only:
```js
function a() {return 'a';}
a();
```

in bigger projects, huge amounts of unnecessary code can be bundled in.  

is there an easy way to fix this?

**Comment by kitsonk** [comment]
@max-pub what version of Deno are you using?

cc/ @kdy1

**Comment by kdy1** [comment]
Fixed on upstream.

**Comment by bartlomieju** [comment]
Fixed in 1.8
